hearthstone/simulator/core/spell_pool.py

```diff
@@ -81,7 +81,8 @@ class MightOfStormwind(Spell):
 
     def on_play(self, context: 'BuyPhaseContext', board_index: Optional['BoardIndex'] = None,
                 store_index: Optional['StoreIndex'] = None):
-        for card in context.owner.in_play:
+        for _ in range(3):  # TODO: is this three different random minions?
```

Comment:
No, this could select the same minions.  You need to create a copy of the list (make sure you are not just getting another reference to the in_play list), and then remove the elements from that copy when they are selected.

Reply:
This is a question about the in-game behavior!